src/framework/scene-registry.js

```diff
@@ -0,0 +1,286 @@
+pc.extend(pc, function () {
+
+    /**
+     * @private
+     * @name pc.SceneRegistryItem
+     * @class Item to be stored in the {@link pc.SceneRegistry}
+     * @param {String} name The name of the scene
```

Comment:
Users can rename scenes in the Editor. Do we need to handle it?

Reply:
Changing scene name in the editor requires a page refresh to be picked up